products/green_fast_track/models/product/sources/sources__arterial_highways.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
+    variable_type,
+    variable_id,
+    ST_MULTI(raw_geom) AS raw_geom
+FROM {{ ref('stg__dcm_arterial_highways') }}
```

Comment:
See [stg table](https://github.com/NYCPlanning/data-engineering/blob/22feb515943b8e9cec64df8d8135923b7441964c/products/green_fast_track/models/staging/stg__dcm_arterial_highways.sql) and [int table](https://github.com/NYCPlanning/data-engineering/blob/22feb515943b8e9cec64df8d8135923b7441964c/products/green_fast_track/models/intermediate/buffers/int_buffers__dcm_arterial_highways.sql)

Reply:
thanks! the pre-commit checks caught the other 2 i had to change but not this one because `stg__dcm_arterial_highways` still exists and I didn't re-run a build